.github/actions/calculate-docker-image/action.yml

```diff
@@ -86,6 +96,7 @@ runs:
       env:
         IMAGE_NAME: ${{inputs.docker-image-name}}
         DOCKER_SKIP_S3_UPLOAD: "1"
+        DOCKER_SKIP_PUSH: ${{ steps.check.outputs.skip_push || 'false' }}
```

Comment:
Setting a default to `false` here since we'd like to keep the legacy behavior here and only want to skip push in the case of the stampeding herd example